test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -296,8 +296,71 @@ function ExecuteWebRequest
     return $result
 }
 
+[string] $verboseEncodingPrefix = 'Content encoding: '
+# This function calls Invoke-WebRequest with the given uri and
+# parses the verbose output to determine the encoding used for the content.
+function ExecuteRestMethod
+{
+    param
+    (
+        [Parameter(Mandatory)]
+        [string]
+        $Uri,
+
+        [switch] $UseBasicParsing
+    )
+    $result = [PSObject]@{Output = $null; Error = $null; Encoding = $null; Content = $null}
```

Comment:
i think that '[PSObject]' should probably be '[PSCustomObject]'  - the `[PSObject]` doesn't actually convert the type away from a hashtable. So, `[PSObject]` can be removed, or changed to `[PSCustomObject]`

Reply:
Will do.

Reply:
Fixed